src/app/shared/interfaces/http-author.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface HttpAuthor {
```

Comment:
Basically you do not name you interfaced with the _http_

Reply:
yes, I know, I named them in such way because there were made interfaces for user, courses etc. in the first homeworks.
But they don't meet the requirements for interfaces from server side (there were other fields). So I renamed new correct interfaces with prefixes http